activemodel/lib/active_model/validations/exclusion.rb

```diff
@@ -29,7 +29,9 @@ module HelperMethods
       # Configuration options:
       # * <tt>:in</tt> - An enumerable object of items that the value shouldn't
       #   be part of. This can be supplied as a proc, lambda or symbol which returns an
-      #   enumerable. If the enumerable is a range the test is performed with
+      #   enumerable. If the enumerable is a numerical, time or datetime range the test
+      #   is performed with <tt>Range#cover?</tt>, otherwise with <tt>include?</tt>. When
```

Comment:
According to the [docs](http://edgeguides.rubyonrails.org/api_documentation_guidelines.html#fixed-width-font), please use `+:include?+`.